Compiler warnings

Compiling fc7847e on Arch Linux there are some compiler warnings
```
[...]
    Ignored 63 untranslated source text(s)
Scanning dependencies of target pavucontrol-qt
[ 74%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/pavucontrol.cc.o
[ 76%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/mainwindow.cc.o
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/mainwindow.cc: In constructor 'MainWindow::MainWindow()':
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/mainwindow.cc:100:13: warning: unused variable 'width' [-Wunused-variable]
         int width  = g_key_file_get_integer(config, "window", "width", nullptr);
             ^~~~~
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/mainwindow.cc:101:13: warning: unused variable 'height' [-Wunused-variable]
         int height = g_key_file_get_integer(config, "window", "height", nullptr);
             ^~~~~~
[ 78%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/cardwidget.cc.o
[ 80%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/channelwidget.cc.o
[ 82%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/devicewidget.cc.o
[ 84%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/minimalstreamwidget.cc.o
In file included from /usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/devicewidget.cc:28:0:
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/devicewidget.h: In constructor 'DeviceWidget::DeviceWidget(MainWindow*, QByteArray)':
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/devicewidget.h:88:16: warning: 'DeviceWidget::mDeviceType' will be initialized after [-Wreorder]
     QByteArray mDeviceType;
                ^~~~~~~~~~~
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/devicewidget.h:49:10: warning:   'bool DeviceWidget::offsetButtonEnabled' [-Wreorder]
     bool offsetButtonEnabled;
          ^~~~~~~~~~~~~~~~~~~
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/devicewidget.cc:33:1: warning:   when initialized here [-Wreorder]
 DeviceWidget::DeviceWidget(MainWindow* parent, QByteArray deviceType) :
 ^~~~~~~~~~~~
[ 86%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/rolewidget.cc.o
[ 88%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/sinkinputwidget.cc.o
[ 90%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/sinkwidget.cc.o
[ 92%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/sourceoutputwidget.cc.o
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/sinkwidget.cc: In member function 'virtual void SinkWidget::executeVolumeUpdate()':
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/sinkwidget.cc:80:10: warning: unused variable 'dev' [-Wunused-variable]
     char dev[64];
          ^~~
/usr/local/peter/Tests/pavucontrol-qt/Projekt/pavucontrol-qt/src/sinkwidget.cc:81:9: warning: unused variable 'playing' [-Wunused-variable]
     int playing = 0;
         ^~~~~~~
[ 94%] Building CXX object src/CMakeFiles/pavucontrol-qt.dir/sourcewidget.cc.o
[...]
```
Couldn't see any malfunction due to them. Yet I assume they're worth having another look at.

--- a/src/devicewidget.cc
+++ b/src/devicewidget.cc
@@ -33,9 +33,9 @@
 /*** DeviceWidget ***/
 DeviceWidget::DeviceWidget(MainWindow* parent, QByteArray deviceType) :
     MinimalStreamWidget(parent),
+    offsetButtonEnabled(false),
     mpMainWindow(parent),
-    mDeviceType(deviceType),
-    offsetButtonEnabled(false) {
+    mDeviceType(deviceType) {
 
     setupUi(this);
     advancedWidget->hide();

--- a/src/mainwindow.cc
+++ b/src/mainwindow.cc
@@ -106,12 +106,11 @@ MainWindow::MainWindow():
         if (g_key_file_has_key(config, "window", "showVolumeMeters", nullptr)) {
             showVolumeMetersCheckButton->setChecked(g_key_file_get_boolean(config, "window", "showVolumeMeters", nullptr));
         }
-#if 0
-        int default_width, default_height;
-        get_default_size(default_width, default_height);
-        if (width >= default_width && height >= default_height)
+
+        const QSize default_size = size(); // widget is resized in setupUi based on values in *.ui
+        if (width >= default_size.width() && height >= default_size.height())
             resize(width, height);
-#endif
+
         int sinkInputTypeSelection = g_key_file_get_integer(config, "window", "sinkInputType", &err);
         if (err == nullptr)
             sinkInputTypeComboBox->setCurrentIndex(sinkInputTypeSelection);
@@ -684,7 +683,7 @@ void MainWindow::updateSinkInput(const pa_sink_input_info &info) {
 
     if ((t = pa_proplist_gets(info.proplist, "module-stream-restore.id"))) {
         if (strcmp(t, "sink-input-by-media-role:event") == 0) {
-            g_debug(tr("Ignoring sink-input due to it being designated as an event and thus handled by the Event widget").toUtf8().constData());
+            g_debug("%s", tr("Ignoring sink-input due to it being designated as an event and thus handled by the Event widget").toUtf8().constData());
             return;
         }
     }

--- a/src/pavucontrol.cc
+++ b/src/pavucontrol.cc
@@ -580,7 +580,7 @@ void context_state_callback(pa_context *c, void *userdata) {
             context = NULL;
 
             if (reconnect_timeout > 0) {
-                g_debug(QObject::tr("Connection failed, attempting reconnect").toUtf8().constData());
+                g_debug("%s", QObject::tr("Connection failed, attempting reconnect").toUtf8().constData());
                 g_timeout_add_seconds(reconnect_timeout, connect_to_pulse, w);
             }
             return;
@@ -631,7 +631,7 @@ gboolean connect_to_pulse(gpointer userdata) {
                 reconnect_timeout = -1;
                 qApp->quit();
             } else {
-                g_debug(QObject::tr("Connection failed, attempting reconnect").toUtf8().constData());
+                g_debug("%s", QObject::tr("Connection failed, attempting reconnect").toUtf8().constData());
                 reconnect_timeout = 5;
                 g_timeout_add_seconds(reconnect_timeout, connect_to_pulse, userdata);
             }

--- a/src/sinkwidget.cc
+++ b/src/sinkwidget.cc
@@ -77,33 +77,12 @@ SinkWidget::SinkWidget(MainWindow *parent) :
 
 void SinkWidget::executeVolumeUpdate() {
     pa_operation* o;
-    char dev[64];
-    int playing = 0;
-
     if (!(o = pa_context_set_sink_volume_by_index(get_context(), index, &volume, NULL, NULL))) {
         show_error(tr("pa_context_set_sink_volume_by_index() failed").toUtf8().constData());
         return;
     }
 
     pa_operation_unref(o);
-#if 0
-    ca_context_playing(ca_gtk_context_get(), 2, &playing);
-    if (playing)
-        return;
-
-    snprintf(dev, sizeof(dev), "%lu", (unsigned long) index);
-    ca_context_change_device(ca_gtk_context_get(), dev);
-
-    ca_gtk_play_for_widget(GTK_WIDGET(gobj()),
-                           2,
-                           CA_PROP_EVENT_DESCRIPTION, tr("Volume Control Feedback Sound").toUtf8().constData(),
-                           CA_PROP_EVENT_ID, "audio-volume-change",
-                           CA_PROP_CANBERRA_CACHE_CONTROL, "permanent",
-                           CA_PROP_CANBERRA_ENABLE, "1",
-                           NULL);
-
-    ca_context_change_device(ca_gtk_context_get(), NULL);
-#endif
 }
 
 void SinkWidget::onMuteToggleButton() {

